src-electron/db/query-config.js

```diff
@@ -245,7 +245,7 @@ INTO ENDPOINT_TYPE_ATTRIBUTE (
       cluster.endpointTypeClusterId,
       attributeId,
       staticAttribute.defaultValue ? staticAttribute.defaultValue : '',
-      dbEnum.storageOption.ram,
+      staticAttribute.storagePolicy,
```

Comment:
What is staticAttribute?

Reply:
it's an object returned from a query that retrieves attribute data